src/BasePaintRewards.sol

```diff
@@ -16,20 +22,21 @@ contract BasePaintRewards is Ownable(msg.sender), ERC20("BasePaint Rewards", "BP
     mapping(address referrer => uint256 bips) public rewardRate; // bpis, 1 = 0.1%
     uint256 public defaultRewardRate = 10; // 1.0%
 
+    error NotEnoughContractFunds();
+    error NoRewards();
+    error TransferFailed();
+    error InvalidRate();
+
     event ToppedUp(uint256 amount);
 
     constructor(IBasePaint _basepaint) {
         basepaint = _basepaint;
     }
 
-    function mintLatest(address to, uint256 count, address sendRewardsTo) public payable {
-        uint256 day = basepaint.today() - 1;
-        mint(day, to, count, sendRewardsTo);
-    }
-
-    function mint(uint256 tokenId, address to, uint256 count, address sendRewardsTo) public payable {
+    function mint(address sendMintsTo, uint256 count, address sendRewardsTo) public payable {
```

Comment:
We have to keep "mint latest" and "mint a specific token" separate to make sure affiliates who track per-token mints can continue supporting every BasePaint day